www/js/components/map-page.js

```diff
@@ -16,9 +16,21 @@ class MapPage extends Component {
 
   render() {
     const { marker, services, alerts } = this.props;
-    let selectedPoint = services[0];
-    if (marker.selectedMarker) {
+
+    let selectedPoint;
+    if(marker.selectedMarker) {
       selectedPoint = marker.selectedMarker;
+    } else if(services.length > 0) {
+      selectedPoint = services[0];
+    } else {
+      selectedPoint = { };
+    }
+
```

Comment:
The UI now renders before any services are loaded from the DB. I am not sure how the point map and point card work with a null `selectedPoint`